e2e/specs/st_main_menu.spec.js

```diff
@@ -23,22 +23,49 @@ describe("main menu", () => {
     cy.get("[data-testid='stDecoration']").invoke("css", "display", "none");
   });
 
-  it("displays menu dropdown", () => {
+  it("displays light main menu and about section properly", () => {
     cy.get("[data-testid='stConnectionStatus']").should("not.exist");
 
     // Main menu renders visually as we expect
     cy.get("#MainMenu > button").click();
+
     // Cypress cuts the popover off due to the transform property, so we move
     // the main menu to a location to show it clearly for snapshots.
     cy.get('[data-testid="main-menu-popover"]').invoke(
       "attr",
       "style",
       "transform: translate3d(20px, 20px, 0px)"
     );
-    cy.get('[data-testid="main-menu-list"]').matchImageSnapshot("main_menu");
```

Comment:
You might need to keep this code in the snapshots for dark mode since the added dark main menu snapshots look weird compared to the light mode one (they're partially cut off)